scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/DatasetController.java

```diff
@@ -0,0 +1,191 @@
+package edu.asu.diging.scriptoocloud.web;
+
+import edu.asu.diging.scriptoocloud.core.exceptions.DatasetException;
+import edu.asu.diging.scriptoocloud.core.model.IDataset;
+import edu.asu.diging.scriptoocloud.core.service.IDatasetService;
+import edu.asu.diging.scriptoocloud.core.service.impl.DataFileService;
+import edu.asu.diging.scriptoocloud.core.service.impl.DatasetService;
+import edu.asu.diging.scriptoocloud.web.forms.DatasetForm;
+import edu.asu.diging.simpleusers.core.model.IUser;
+import edu.asu.diging.simpleusers.core.service.IUserManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+import java.util.List;
+
+
+@Controller
+public class DatasetController {
+
+    private final DatasetService datasetService;
+    private final DataFileService dataFileService;
+    private final IDatasetService datasetManager;
+    private final IUserManager userManager;
+
+    @Autowired
+    public DatasetController(DatasetService datasetService,
+                             DataFileService dataFileService,
+                             IDatasetService datasetManager,
+                             IUserManager userManager) {
+        this.datasetService = datasetService;
+        this.dataFileService = dataFileService;
+        this.datasetManager = datasetManager;
+        this.userManager = userManager;
+    }
+
+    @RequestMapping(value = "datasets/{username}", method = RequestMethod.GET)
```

Comment:
this should just be `/datasets`, the username should come from the principal (or anyone can access anyone else's datasets)